pkg/types/change/device/types.proto

```diff
@@ -97,14 +97,25 @@ message TypedValue {
     repeated int32 type_opts = 3  [json_name="TypeOpts", (gogoproto.jsontag) = "TypeOpts,omitempty"];
 }
 
-// Value is an individual Path/Value combination in a Change
+// Value is an individual Path/Value and removed flag combination in a Change
 message Value {
     // 'path' is the path to change
-    string path = 1;
+    string path = 1 [json_name="Path", (gogoproto.jsontag) = "Path,omitempty"];
 
     // 'value' is the change value
-    TypedValue value = 2;
+    TypedValue value = 2 [json_name="Value", (gogoproto.jsontag) = "Value,omitempty"];
 
     // 'removed' indicates whether this is a delete
     bool removed = 3 [(gogoproto.jsontag) = "Remove,omitempty"];
 }
+
+// PathValue is an individual Path/Value combination - it is like Value above
+// without the removed flag - it is not used in the DeviceChange store
+// Instead it is useful for handling OpState where `removed` is not relevant
+message PathValue {
```

Comment:
Instead of PathValue with is confusing, I think `StateValue` is actually what you mean here. This signifies a Value that is State, meaning Read only no r/w

Reply:
As agreed left `PathValue` as it as it is for more than just State

Instead moved `Value` in types to be `ChangeValue`